#############################################################
#                                                           #
# Copyright (c) 2022, University of California, Los Angeles #
#                                                           #
# Authors: Kenny J. Chen, Brett T. Lopez                    #
# Contact: kennyjchen@ucla.edu, btlopez@ucla.edu            #
#                                                           #
#############################################################

dlo_odom:
  ros__parameters:
    dlo/version: 1.4.3
    dlo/adaptiveParams: true
    dlo/imu: true
    dlo/gravityAlign: true
    dlo/odomNode/odom_frame: odom
    dlo/odomNode/child_frame: base_link

dlo_map:
  ros__parameters:
    publishFullMap: true
    publishFreq: 1.0
    leafSize: 0.25
    odom_frame: odom
